src/FileUtility.ts

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+import * as fs from 'fs-plus';
+import * as path from 'path';
+import * as sdk from 'vscode-iot-device-cube-sdk';
+import {ScaffoldType} from './constants';
+
+export class FileUtility {
+
+  static async exists(type: ScaffoldType, path: string): Promise<boolean> {
```

Comment:
better call it
  static async fileExists(environmentType: EnvironmentType, path: string): Promise<boolean> {

Reply:
This functions can judge not only **file** but also **directory**. So we use `exists` as the function name.
We can change the `type` argument to `environmentType`.

Reply:
But actually, the type is not indicating the environment whether is local or in workspace, since scenario can be that when we are in remote side and want to create a new project. In this situation we will pass a `local` type as argument in order to use the cube api to scaffold files in local filesystem.
i.e. the `type` argument is more about where to scaffold the new files, in local filesystem or in workspace.
Will it be better if we use the `scaffoldType` as the argument name?